src/core.js

```diff
@@ -321,7 +319,7 @@ var CryptoJS = CryptoJS || (function (Math, undefined) {
             var words = [];
 
             for (var i = 0; i < nBytes; i += 4) {
-                words.push((secureRandom() * 0x100000000) | 0);
+                words.push((cryptoSecureRandomInt());

```

Comment:
That's a synax error, you forgot to remove a `(`.

Reply:
Right, I did not run the code yet ...